docs/source/api/api.rst

```diff
@@ -606,4 +606,145 @@ Example: GET /locations/
           "name": "Newport"
         }
       ]
-    }
\ No newline at end of file
+    }
+
+Themes listing
+----------------
+
+The themes listing is available at ``/themes/``. It returns a JSON array
+consisting of each of the themes, and information about them.
+The colors selected are sent as a RGB string.
+For activating a particular theme set its ``Active`` field as ``Yes``.
```

Comment:
This is the list endpoint, so it's read-only. The active theme has its ``active`` field set to ``Yes``.